### DISCUSSION TITLE
predicting voltage knee point near end

### DISCUSSION THREAD

**Discussion by Thiyaga2025** [pybamm-team/PyBaMM#4482] [Q&A]
team pybamm,

i am trying to run simulations with OKane2022 parameter set for an NMC cell at c/3 discharge rate. I have difficulty matching the knee point in voltage curve near discharge ending. what property/parameter i should tune to get the knee point in pybamm better. thanks for the help.

![image](https://github.com/user-attachments/assets/cce706d0-12ab-40a5-a936-a84ce7ac4520)

**Comment by DrSOKane** [comment]
Where does the data labelled "test" come from?

**Reply by Thiyaga2025** [reply]
@DrSOKane,

thanks a ton for the response. that is from the cycler, lab data using reference performance test(RPT).

**Reply by DrSOKane** [reply]
Do you have any low current tests, such as C/10? That will determine whether the issue is with with the open-circuit voltage or with the dynamic parameters such as diffusivity.

**Reply by Thiyaga2025** [reply]
@DrSOKane ,

i don't have low current test data now. your suggestion is very valid. the cathode has got a minor blending. will try to get OCV data if that does not work then will try changing diffusivity, etc.,
thanks.

**Reply by DrSOKane** [reply]
If you don't have any low current test data, try using `plot_voltage_components()`, like in [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/plotting/plot-voltage-components.html).

**Reply by Thiyaga2025** [reply]
@DrSOKane ,

please let me know your thoughts. thanks.

![image](https://github.com/user-attachments/assets/5ed1ba8d-0383-4dfd-9d70-007666397009)

**Reply by DrSOKane** [reply]
Looks like the open-circuit potential is the culprit. The OKane2022 parameters are for NMC811, so if you're using a different type of NMC, you will need a different positive electrocde open-circuit potential.

**Reply by Thiyaga2025** [reply]
@DrSOKane ,

thanks for the valuable insight. will try get the OCV curve for the blend electrode.
